packages/babel-preset/common.js

```diff
@@ -0,0 +1,100 @@
+const nonStandardPlugins = require('./non-standard-plugins');
+
+module.exports = function shopifyCommonPreset(api, options = {}) {
+  const {
+    targets = {
```

Comment:
I think we should push towards only browserslist strings as targets rather than using the object notation. `current node` is a valid browserlist target.

Pending comments below I think we remove the need for `isNode` and thus for this to be an object